src/controllers/equipmentController.js

```diff
@@ -0,0 +1,14 @@
+class EquipmentController {
+  constructor(equipmentService) {
+    this.equipmentService = equipmentService;
+    this.use = this.use.bind(this);
+  }
+
+  use(req, res) {
+    this.equipmentService.use(req.body)
+      .then(data => res.status(200).json(data))
+      .catch(err => res.status(err.message).json('no luck'));

```

Comment:
this seems off, the status gets the message, but that should be a number, no? also, please return a more meaningful error statement.

Reply:
err.message is from global Error object, the message is a number, similar to RegistrationController